pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -305,6 +305,13 @@
     )
     private long topicLoadTimeoutSeconds = 60;
 
+    @FieldContext(
+            category = CATEGORY_SERVER,
```

Comment:
Should be `CATEGORY_AUTHENTICATION` ?

Reply:
I thought about it, and I thought the decryption class should be a generic configuration.It is required for sensitive configurations, not only for authorization.

Reply:
Ok, make sense.

Reply:
@codelipenghui thanks for you review , do you have any other suggestions? and I also commit  a pr to fix zookeeper promethus connetion data #9777 , can you please take a look? thanks.

I will try to commit some bug fix 、security Improvements and bookkeeper SASL-SCRAM later
Hope can be merge to pulsar ,so we can upgrade new pulsar version directly ,no need to rebuild after custom-made